pocs/scheduler/observation.py

```diff
@@ -127,6 +125,15 @@ def directory(self):
                                            self.field.field_name)
             return self._directory
 
+    @property
+    def current_exp_num(self):
+        """ Return the current number of exposures.
```

Comment:
Style question: why the leading space before the start of the docstring contents (i.e. the space before Return).

Reply:
I've not seen that style before. Where does it come from? It it widely used in the python community? In astropy?

Reply:
And looking at PEP8 (https://www.python.org/dev/peps/pep-0008/?#documentation-strings), it doesn't use that style:
```
PEP 257 describes good docstring conventions. Note that most importantly, the """ that ends a multiline docstring should be on a line by itself:

"""Return a foobang

Optional plotz says to frobnicate the bizbaz first.
"""
```

I followed the link to PEP 257, which also doesn't contain any example of a leading space.

Reply:
I use this: https://packagecontrol.io/packages/AutoDocstring

Makes docstring maintenance much nicer.

Reply:
Yeah, I checked my [usual resource](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) and don't see it anywhere. I'll make sure it's gone going forward.